SingletonPattern.java

```diff
@@ -0,0 +1,26 @@
+package Ex4;
```

Comment:
This is very basic. Could you please implement any usecase for singleton pattern?